src/style.css

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+div[class*="stars--"] {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
+
+div[class*="stars--"] .stars__star {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.stars--0 .stars__star.stars__star:nth-last-child(n+1) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.stars--1 .stars__star.stars__star:nth-last-child(-n+4) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.stars--2 .stars__star.stars__star:nth-last-child(-n+3) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.stars--3 .stars__star.stars__star:nth-last-child(-n+2) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
```

Comment:
Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)